scripts/openroad/replace.tcl

```diff
@@ -66,15 +66,17 @@ if { $::env(PL_BASIC_PLACEMENT) } {
 }
 
 if { $::env(PL_TIME_DRIVEN) } {
+	source $::env(SCRIPTS_DIR)/openroad/set_rc.tcl
 	read_sdc $::env(CURRENT_SDC)
 	read_verilog $::env(synthesis_results)/$::env(DESIGN_NAME).v
```

Comment:
While it isn't part of your change I need to ask why is the verilog being read here after the DEF has already been loaded?  @donn

Reply:
On that topic - I wonder if it would be better to load everything at the start unconditionally (perhaps with a helper script), vs loading things as we need them, based on config variables. As an example, we did call set_rc.tcl, just later on in the script.

Reply:
https://github.com/The-OpenROAD-Project/OpenLane/pull/1041 is a first step to be more consistent in how we load files. tech liberty, other liberty files, lef, def.

Reply:
@maliberty Pre-dates me. My git blame is just me updating an environment variable.

Reply:
@donn can you remove the read_verilog and see if that has any effect?

Reply:
I removed a few unncessary calls to `read_verilog` in https://github.com/The-OpenROAD-Project/OpenLane/pull/1051

Reply:
Removed.